_pages/code.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: main
+title: Code
+slogan: Open source projects...
+theme: light
+permalink: /code/
+css: 
+    - normalize
```

Comment:
Every page uses `normalize.css` and `simple-grid.min.css` so I'd recommend just linking to these in `_includes/head.html` and using the `css` attribute here for additional css files for this page.